src/Troubleshooting/index.md

```diff
@@ -67,19 +51,16 @@ This issue has no known cause, but appears to be more frequent with the Vinegar
 
 ### Roblox doesn't launch from the website
```

Comment:
This can be removed, Vinegar automatically sets MIME associations upon Studio installation.